portal-web/docroot/html/taglib/aui/button/end.jsp

```diff
@@ -47,7 +47,8 @@
 					onClick="<%= onClick %>"
 				</c:when>
 				<c:when test="<%= Validator.isNotNull(escapedHREF) %>">
-					onClick="Liferay.Util.navigate('<%= escapedHREF %>')"
+					data-href="<%= escapedHREF %>"
+					onClick="Liferay.Util.navigate(this.getAttribute('data-href'))"
```

Comment:
[nit] We can simplify this a bit, to `Liferay.Util.navigate(this.dataset.href)`. 

[even smaller nit] It's a bit odd to see camelCase `onclick` in raw HTML, but it seems browsers have been fine with it for a long time.

Reply:
I've force pushed the first proposal. Second one would require a deeper review of `onClick` occurrences